pages/ksphere/kommander/1.1.0-beta/projects/addon-catalog/index.md

```diff
@@ -25,9 +25,19 @@ spec:
   url: https://github.com/mesosphere/kubeaddons-enterprise
 ```
 
-To deploy an addon, select a Project from your Workspace and select the **View Catalog** button to browse the available addons from your configured repositories.
+To deploy an addon:
 
-To customize an addon before deploying it, you can customize the helm chart values or KUDO parameters of a service before deploying it.
+1. Select **Workspace > Project** 
```

Comment:
I might be missing something, but all steps are step 1. Also, we don't mention creating a project.